ktxspec.adoc

```diff
@@ -879,59 +879,110 @@ standard gamut images.
   `KHR_DF_TRANSFER_LINEAR`.
 
 ==== Providing additional information
-There is one case where the _dfDescriptorBlock_ is used to
+There are several cases where the _dfDescriptorBlock_ is used to
 provide information beyond that given by <<_vkformat,vkFormat>>.
 
 Premultiplied Alpha::
   `KHR_DF_FLAG_ALPHA_PREMULTIPLIED` (= 1) can be set in the `flags`
   field if the images' RGB components have been multiplied by their
   alpha components, otherwise it must be 0.
 
-==== Special Cases
-
-===== Universal ASTC Format
-The Universal ASTC image format (UASTC) can be specified using
-`colorModel` `KDF_DF_MODEL_UASTC` (= 166) together with `vkFormat`
-`VK_FORMAT_UNDEFINED` (= 0). The DFD must be as described in Section
-5.6.14 "KHR_DF_MODEL_UASTC" of <<KDF13>>. Images in this format
-must be transcoded to a GPU-supported block-compressed format before
-being uploaded to and sampled by a GPU. UASTC images can be
-supercompressed with Zstandard (`supercompressionScheme` = 2) with
-or without first conditioning the data with Rate Distortion
-Optimization. If supercompression is used, the DFD's `bytesPlane[0-7]`
-must be set to 0 as described in the next subsection.
-
+Basis Universal UASTC Format::
+The Universal ASTC image format (UASTC) is indicated by `colorModel`
+`KDF_DF_MODEL_UASTC` (= 166) together with `vkFormat` `VK_FORMAT_UNDEFINED`
+(= 0). The DFD must be as described in Section 5.6.14 "KHR_DF_MODEL_UASTC"
+of <<KDF13>>. Images in this format must be transcoded to a
+GPU-supported block-compressed format before being uploaded to and
```

Comment:
> Images in this format must be transcoded to a GPU-supported block-compressed format

or decoded to a GPU-supported uncompressed format

Reply:
Done.